tests/stream/test_stream_smoke/0013_changelog_stream3.json

```diff
@@ -144,26 +144,22 @@
-                        {"client":"python", "query_type": "table", "depends_on":"1455", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'a', 11.1)"},
-                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (2, 'a', 33.3)"},
-                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'b', 22.2)"},
-                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (2, 'b', 22.2)"},
-                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'c', 33.3)"},
-                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (2, 'c', 11.1)"},
-                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'b', 44.4)"},
-                        {"client":"python", "query_type": "table", "kill":"1455", "kill_wait":3, "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (2, 'b', 44.4)"}
+                        {"client":"python", "query_type": "stream", "query_id":"1455", "wait":1, "terminate":"manual", "query":"with subquery as (select id, count() as cnt, min(val) as min_val, max(val) as max_val from changelog(test14_append_stream_3, name) group by id emit periodic 1s) select count(*), sum(cnt), min(min_val), max(max_val) from subquery"},
+                        {"client":"python", "query_type": "table", "depends_on":"1455", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'a', 11.1)(2, 'a', 33.3)"},
+                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'b', 22.2)(2, 'b', 22.2)"},
+                        {"client":"python", "query_type": "table", "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'c', 33.3)(2, 'c', 11.1)"},
+                        {"client":"python", "query_type": "table", "kill":"1455", "kill_wait":3, "wait":1, "query": "insert into test14_append_stream_3 (id, name, val) values (1, 'b', 44.4)(2, 'b', 44.4)"}
                     ]
                 }
             ],
             "expected_results": [
                 {
                     "query_id":"1455",
                     "expected_results":[
-                        [2, 1, 0.0, 33.29999923706055],
-                        [2, 2, 0.0, 33.29999923706055],
-                        [2, 3, 0.0, 33.29999923706055],
-                        [2, 3, 0.0, 44.400001525878906]
+                        [1, 1, 33.3, 33.3],
```

Comment:
What cause the expected results changes

Reply:
This test case is unstable and counts a deleted group (i.e. empty default value), so I modified it.